### PR TITLE
Domains: Add notice for specific TLDs requiring HSTS

### PR DESCRIPTION
**Pull request by aidvu**
#### Changes proposed in this Pull Request

We're onboarding new TLDs soon, and they require some
UI changes. Show notice for HSTS requirement now.

#### Testing instructions
- Hack `isHstsMandatory` to `return true;` for a specific TLD
- Check domain search results
- Check for all screen sizes

Dependency: D27031-code

### CODE REVIEW

File: client/components/info-popover/index.jsx [hunk 1]
```diff
@@ -50,6 +50,8 @@ export default class InfoPopover extends Component {
 
 	handleClick = event => {
 		event.preventDefault();
+		event.stopPropagation();
```
Comment (aidvu) [inline client/components/info-popover/index.jsx]: Don't think there exists a case when we want both the popover to show and to submit another action. :)

**Comment by aidvu** [comment]
@hambai you want HSTS info on checkout or both? 

As for icon, I like this one better.

**Comment by hambai** [comment]
> @hambai you want HSTS info on checkout or both?
I think it makes sense to have the notification in both places, since the info popover is something that requirese user action to show the text, but if we show it in the checkout too there is no chace of anyone missing it.

> As for icon, I like this one better.
Show me one place where we display the info popover icon like that :D It is incosistent with every other place I found where we show the info icon... See:
Screenshot 2019-04-22 at 12 06 17
Screenshot 2019-04-22 at 12 03 12
Screenshot 2019-04-22 at 12 02 48

**Comment by aidvu** [comment]
Ahh, consistency is important. :) Will update.

**Pull request merged**